includes/filerepo/LocalRepo.php

```diff
@@ -194,16 +194,25 @@ protected function getArticleID( $title ) {
 		if( !$title instanceof Title ) {
 			return 0;
 		}
-		$dbr = $this->getSlaveDB();
-		$id = $dbr->selectField(
-			'page', // Table
-			'page_id',  //Field
-			array(  //Conditions
-				'page_namespace' => $title->getNamespace(),
-				'page_title' => $title->getDBkey(),
-			),
-			__METHOD__  //Function name
-		);
+
+		// Wikia change - begin
+		try {
+			$dbr = $this->getSlaveDB();
+			$id = $dbr->selectField(
+				'page', // Table
+				'page_id',  //Field
+				[  //Conditions
+					'page_namespace' => $title->getNamespace(),
+					'page_title' => $title->getDBkey(),
+				],
+				__METHOD__  //Function name
+			);
+		} catch ( DBError $e ) {
```

Comment:
DBError has logging implemented in constructor, do you think additional logging is needed here?